radicle-httpd/src/api/v1/projects.rs

```diff
@@ -401,6 +440,43 @@ async fn issue_handler(
     Ok::<_, Error>(Json(issue))
 }
 
+/// Returns the name part of a path string.
+fn name_in_path(path: &str) -> &str {
+    match path.rsplit('/').next() {
+        Some(name) => name,
+        None => path,
+    }
+}
+
+/// Returns JSON of a commit.
+fn commit_json(commit: &Commit) -> serde_json::Value {
+    json!({
+      "sha1": commit.id,
+      "author": {
+        "name": commit.author.name,
+        "email": commit.author.email
+      },
+      "summary": commit.summary,
+      "description": commit.description(),
+      "committer": {
+        "name": commit.committer.name,
+        "email": commit.committer.email
+      },
+      "committerTime": commit.committer.time.seconds()
+    })
+}
+
+/// Returns JSON for a blob with a given `path`.
+fn blob_json(blob: &Blob, path: &str) -> serde_json::Value {
+    json!({
+        "binary": blob.is_binary(),
+        "content": blob.content(),
+        "name": name_in_path(path),
+        "path": path,
+        "lastCommit": commit_json(blob.commit())
+    })
+}
```

Comment:
Can you move these two functions to `json::commit` and `json::blob` by creating a `json` module in here?

Reply:
Will do.

Reply:
put it into its own file `/api/json.rs`

Reply:
Done.